R/FilterStateRange.R

```diff
@@ -457,10 +457,40 @@ RangeFilterState <- R6::R6Class( # nolint
       tagList(
         div(
           class = "choices_state",
+          tags$head(tags$script(
```

Comment:
This script suddenly grew to a substantial size. Please ad a short comment explaining why it has to be inline rather than in a separate .js file.

Reply:
added comment.